syntheseus/search/algorithms/base.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         self,
         limit_iterations: int = INT_INF,
         limit_reaction_model_calls: int = INT_INF,
+        limit_graph_nodes: int = INT_INF,
```

Comment:
Do we also want to have this as an option in the search CLI?

Reply:
Good idea. I just pushed another commit to do this. Can you check whether you think I've modified the file in all the right places?

Reply:
(@kmaziarz , sorry forgot to tag you)